### PR TITLE
Add functions for integer monotonic ticks

### PR DESCRIPTION
**Pull request by Wallbraker**
This lets you get raw ticks from ohmd and convert to whichever rate you want.

Cheers, Jakob.

### CODE REVIEW

**Comment by Wallbraker** [comment]
Testing!

File: include/openhmd.h [hunk 1]
```diff
@@ -13,6 +13,8 @@
 #ifndef OPENHMD_H
 #define OPENHMD_H
 
+#include "stdint.h"
```
Comment (noname22) [inline include/openhmd.h]: <stdint.h> ?

Comment (Wallbraker) [reply include/openhmd.h]: Oh right, thanks!

**Comment by TheOnlyJoey** [comment]
LGTM!

**Pull request merged**